Assets/MixedRealityToolkit/_Core/Definitions/Devices/ControllerMappingLibrary.cs

```diff
@@ -224,10 +225,588 @@ public static class ControllerMappingLibrary
             new InputManagerAxis { Name = AXIS_27, Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 27 }
         };
 
-#endif
+        #endregion InputAxisConfig
+
+        #region Controller Image Resources
+
+        private static Texture2D xboxControllerWhite;
+
+        public static Texture2D XboxControllerWhite
+        {
+            get
+            {
+                if (xboxControllerWhite == null)
+                {
+                    xboxControllerWhite = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/XboxController_white.png", typeof(Texture2D));
+                }
+
+                return xboxControllerWhite;
+            }
+        }
+        private static Texture2D xboxControllerWhiteScaled;
+
+        public static Texture2D XboxControllerWhiteScaled
+        {
+            get
+            {
+                if (xboxControllerWhiteScaled == null)
+                {
+                    xboxControllerWhiteScaled = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/XboxController_white_scaled.png", typeof(Texture2D));
+                }
+
+                return xboxControllerWhiteScaled;
+            }
+        }
+
+        private static Texture2D xboxControllerBlack;
+
+        public static Texture2D XboxControllerBlack
+        {
+            get
+            {
+                if (xboxControllerBlack == null)
+                {
+                    xboxControllerBlack = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/XboxController_black.png", typeof(Texture2D));
+                }
+
+                return xboxControllerBlack;
+            }
+        }
+
+        private static Texture2D xboxControllerBlackScaled;
+
+        public static Texture2D XboxControllerBlackScaled
+        {
+            get
+            {
+                if (xboxControllerBlackScaled == null)
+                {
+                    xboxControllerBlackScaled = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/XboxController_black_scaled.png", typeof(Texture2D));
+                }
+
+                return xboxControllerBlackScaled;
+            }
+        }
+
+        private static Texture2D oculusRemoteControllerWhite;
+
+        public static Texture2D OculusRemoteControllerWhite
+        {
+            get
+            {
+                if (oculusRemoteControllerWhite == null)
+                {
+                    oculusRemoteControllerWhite = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/OculusRemoteController_white.png", typeof(Texture2D));
```

Comment:
I would comment / remove this code (entire oculus block) while awaiting guidance on images

Reply:
No, we should use default images in their place @davidkline-ms as a default until we have agreed new images
Text alone looks darn ugly

Reply:
are these JUST for the mini display? if not, where do the control pointers go to on this image?